src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -32,5 +34,10 @@ public static class TestOptions
 
         public static readonly CSharpCompilationOptions UnsafeDebugDll = DebugDll.WithAllowUnsafe(true);
         public static readonly CSharpCompilationOptions UnsafeDebugExe = DebugExe.WithAllowUnsafe(true);
+
+        public static CSharpCompilationOptions WithStrictMode(this CSharpCompilationOptions options)
+        {
+            return options.WithFeatures(ImmutableArray.Create("strict"));
```

Comment:
As it turns out, a test helper that preserves other feature flags wasn't needed, as there are no other features that we test at the same time.